jira/client.py

```diff
@@ -1750,6 +1792,15 @@ def createmeta(
             Dict[str, Any]
 
         """
+        if self._version >= (9, 0, 0):
+            raise JIRAError(f"Unsupported JIRA version: {self._version}")
```

Comment:
also add the info from below in the exception ... => please use createmeta_issuetypes and createmeta_fieldtypes